packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -221,21 +222,21 @@ $ms-Persona-presenceSizeXl: 28px;
 }
 
 .ms-Persona-details {
-  padding: 0 12px;
-  vertical-align: middle;
+  padding: 0;
+  @include padding-right(24px);
+  @include padding-left(12px);
+  display: flex;
+  flex-direction: column;
+  min-width: 0;
```

Comment:
Ah, looks like I don't need the flex and flex-direction.  Min-width is there to fix truncation (read here https://css-tricks.com/flexbox-truncated-text/ ).